compiler/src/parsing/parser.dyp

```diff
@@ -507,9 +507,13 @@ match_branches :
 match_expr :
   | MATCH lparen expr rparen lbrace match_branches rbrace { Exp.match_ ~loc:(symbol_rloc dyp) $3 $6 }
 
+list_expr_ending :
+  | comma? { None }
+  | comma ELLIPSIS expr { Some $3 }
```

Comment:
Maybe trailing commas after a spread as well? Might not make sense syntactically (just yet) until we support multiple spreads.
```suggestion
  | comma ELLIPSIS expr comma? { Some $3 }
```

Reply:
I don't think it is a good idea to allow a comma at the end of a spread until multi-spread or spreads in non-terminal positions are supported.

I imagine a situation where someone rights code like `[a, b, ...rest,]` and then another dev comes to the code and tries to add `[a, b, ...rest, c]` and it doesn't work.